crate_universe/CONTRIBUTING.md

```diff
@@ -18,16 +19,18 @@ Some areas have unit testing, there are a few (very brittle) integration tests,
 ## How to test local changes
 
 To use a local version, first build it:
+
 ```console
-resolver $ cargo build
+crate_universe $ cargo build
 ```
 
 Then run a bazel build with this environment variable:
+
 ```console
 RULES_RUST_RESOLVER_URL_OVERRIDE=file:///path/to/resolver/target/debug/resolver bazel build //whatever
 ```
 
-To get verbose logging, edit `workspace.bzl` to set `RUST_LOG` to `debug` or `trace` instead of `info`. In particular, that will print out the generated Cargo.toml, and the path to the generated workspace file.
+To get verbose logging, edit `def.bzl` to set `RUST_LOG` to `debug` or `trace` instead of `info`. In particular, that will print out the generated Cargo.toml, and the path to the generated workspace file.
```

Comment:
```suggestion
To get verbose logging, edit `defs.bzl` to set `RUST_LOG` to `debug` or `trace` instead of `info`. In particular, that will print out the generated Cargo.toml, and the path to the generated workspace file.
```